common/src/types/network_order.rs

```diff
@@ -66,6 +67,8 @@ pub struct NetworkOrder {
     /// have `None` in place
     #[serde(skip)]
     pub validity_proof_witnesses: Option<OrderValidityWitnessBundle>,
+    /// The timestamp this order was placed at
```

Comment:
nit: It'd be more accurate to say:
```suggestion
    /// The timestamp this order was received at
```